app/policies/note_policy.rb

```diff
@@ -0,0 +1,21 @@
+class NotePolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  def show?
+    true
+  end
+
+  def create?
+    signed_in?
```

Comment:
```suggestion
    true
```
We can allow this action since we use `authenticate_user!` method for this action in the controller.